exercises/strings/longestWord/longestWord.js

```diff
@@ -7,15 +7,22 @@
  * @returns {string} The longest word in the input string
  */
 
-function longestWord(string) {
-  // This is your job. :)
+function findLongestWord(str) {
+  var words = str.split(' ');
+  var len = 0;
+
+ words.forEach(function(word){
```

Comment:
This works. Curious where you've run into `.forEach` before.

Reply:
Hi Jesse! I was just researching things online and trying things out when I accidentally commit these changes half-way. I don't know how that happened and don't know how to erase this commit. This is not my code or official first draft (I will be re-committing this file when I finish working on it). Sorry about the confusion (the other two commits, isEven and replaceCharacter, I did mean to add).

Reply:
There's no confusion — I was just curious! `.forEach` is yet another way to iterate over an array. You'd use it instead of `for(...)`, `for(... of ...)`, etc.